R/rw.R

```diff
@@ -193,7 +193,7 @@ plot.rw <- function(x, rep = FALSE, ...) {
 #' # in steps of 4 and in replicum B from 4 to 500 in steps of 4.
 #' # To combined the two replicas we have to use
 #'
-#' addstat.rw(rw_replicumB, rw_replicumA, TRUE, FALSE)
+#' #addStat.rw(rw_replicumB, rw_replicumA, TRUE, FALSE)
```

Comment:
I think this and similar verbatim code examples must be encapsulated in `\code{}`

Reply:
Alternatively you can add sample data and the example can actually be run.

Reply:
Thanks, I will do that